Makefile

```diff
@@ -1,4 +1,4 @@
-CONTAINER_WORKDIR=/go/src/github.com/lablup/backend.ai-jail
+CONTAINER_WORKDIR=/backend.ai-jail
```

Comment:
As `/go/src/...` style is no longer required, it is a good idea to simplify the path.
But I'd prefer name it using a general term such as `/src` instead of the package name directly because the containers are for building just our one package only.